docs/release-notes/8.8.asciidoc

```diff
@@ -0,0 +1,95 @@
+* Updates how browser field descriptions are provided to {kib} ({pull}153498[#153498]).
+* Enables multi-level grouping for alerts on the Alerts page, based on various fields ({pull}152862[#152862]).
+* Adds links to the Detection & Response and Entity Analytics dashboards that jump to the Alerts page with filters enabled ({pull}152714[#152714]).
+* Updates the visualizations throughout {elastic-sec} to Lens visualizations ({pull}150531[#150531]).
+* Adds a *Share alert* link to the alert details flyout ({pull}148800[#148800]).
+* Adds a warning message to the Rules page when a maintenance window is running ({pull}155386[#155386]).
+* Adds a global search bar to the Detections and Response and Entity Analytics dashboards ({pull}156832[#156832]).
+
+
+[discrete]
+[[bug-fixes-8.8.0]]
+==== Bug fixes
+
+* Fixes a bug that interfered with the default time range when you opened an alert in Timeline ({pull}156884[#156884]).
+* Updates where the technical preview tags appear for host risk score features ({pull}156659[#156659], {pull}156514[#156514]).
+* Fixes a bug that could cause the Alerts page to become unresponsive after you entered an invalid query ({pull}156542[#156542]).
+* Updates the colors used for entity analytic graphs to match the colors used for alert graphs ({pull}156383[#156383]).
+* Fixes a bug that caused errors on the Data Quality dashboard when a `basePath` was configured ({pull}156233[#156233]).
+* Fixes a bug that could cause problems when different users simultaneously edited a Timeline ({pull}155663[#155663]).
+* Fixes a bug that could cause the wrong number of rules to appear in the modal for duplicating rules ({pull}155959[#155959]).
+* Fixes a bug that could cause a blank option to appear in the Create rule exception form ({pull}155221[#155221]).
+* Fixes issues that affected tags in the Add rule exception component of the Shared Exception Lists page ({pull}155219[#155219]).
+* Fixes a bug that displayed an outdated count of affected rules on the Shared Exception Lists page ({pull}155108[#155108]).
+* Improves performance for rendering indicator match alerts on the Alerts page ({pull}154821[#154821]).
+* Fixes a bug that could affect alert prevalence counts on the Alerts page ({pull}154544[#154544]).
+* Adds the "Add to timeline" action to alert counts on the Detections and Response dashboard and the Entity Analytics dashboard ({pull}154299[#154299]).
+* Fixes a bug that could prevent you from using breadcrumbs to navigate back to the Rules page ({pull}150322[#150322]).
+* Fixes a bug that could prevent the *View all open alerts* button on the Detection and Response dashboard from applying the right filters ({pull}156893[#156893]).
+* Fixes the delete index API so it only removes {elastic-sec} 7.x signals indices (`.siem-signals-<space-id>`), index templates, and ILMs and doesn't delete 8.x alert indices (`.alerts-security.alerts-<space-id>`). 
```

Comment:
@peluja1012 @marshallmain I added this for would you mind reviewing this summary? Also, let me know if we should be linking to a specific PR at the end of the summary. Thanks!

cc: @rylnd

Reply:
Looks good to me! Thanks @nastasha-solomon